script/upload-index-json.py

```diff
@@ -15,16 +16,28 @@
 version = sys.argv[1]
 authToken = os.getenv('META_DUMPER_AUTH_HEADER')
 
+def is_json(myjson):
+  try:
+    json.loads(myjson)
+  except ValueError:
+    return False
+  return True
+
 def get_content(retry_count = 5):
   try:
     request = urllib2.Request(
       BASE_URL + version,
       headers={"Authorization" : authToken}
     )
 
-    return urllib2.urlopen(
+    proposed_content = urllib2.urlopen(
       request
     ).read()
+
+    if is_json(proposed_content):
+      return proposed_content
+    print("bad attempt")
+    raise Exception("Failed to fetch valid JSON from the metadumper service")
```

Comment:
Would it be useful to append the json.loads error message to this for postmortems?

Reply:
@ckerr I'm uncertain as to what that error message may contain (E.g. Secret keys from the metadumper) The error would be logged on the metadumper anyway so we should be 🆗 here without logging